packages/core/src/command/hl7v2-subscriptions/hl7v2-roster-generator.ts

```diff
@@ -45,27 +47,23 @@ export class Hl7v2RosterGenerator {
 
     log(`Running with this config: ${JSON.stringify(loggingDetails)}`);
 
-    const subscribers = await executeWithNetworkRetries(
-      async () => this.getAllSubscribers(states, subscriptions, log),
+    const patients = await executeWithNetworkRetries(
+      async () => this.getAllSubscribedPatients(states, subscriptions),
       {
         maxAttempts: NUMBER_OF_ATTEMPTS,
         initialDelay: BASE_DELAY.asMilliseconds(),
         log,
       }
     );
-    log(`Found ${subscribers.length} total subscribers`);
+    log(`Found ${patients.length} total patients`);
 
-    if (subscribers.length === 0) {
-      const msg = `No subscribers found, skipping roster generation`;
+    if (patients.length === 0) {
+      const msg = `No patients found, skipping roster generation`;
       log(msg);
-      capture.message(msg, {
-        extra: loggingDetails,
-        level: "info",
-      });
-      return;
+      throw new MetriportError(msg, { extra: loggingDetails });
```

Comment:
Think this log can get removed - if we're willing to throw an error then let's just put the message in the error and let our wrapper handle it. 

If there are 0 subscribers for an HIE that's probably a bug as I think to go live with them we'll usually need some kind of sample roster at a minimum, so reasonable to just throw here i think.

```
throw new MetriportError(
  `No patients found, skipping roster generation`,
   { extra: loggingDetails }
);